src/index.ts

```diff
@@ -11,5 +11,6 @@ export {
   WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT,
 } from './utilities/within-content-context';
 export {AppBridgeContext} from './utilities/app-bridge';
+export {ContextualSaveBarProps, ToastProps} from './utilities/frame';
```

Comment:
I dislike the idea of exporting things from utilities within the index, as I think the relationship between what is ToastProps used for is made blury here.

I think what these props are used for is made much clearer if we export `Props` from `src/components/ContextualSaveBar` via the components index - that way the component and its props are kept together.